files/01_setup_http_service.sh

```diff
@@ -12,6 +12,30 @@ set -o pipefail
 #
 # hostname: The hostname of this IPA client (e.g. client.example.com).
 
+USAGE=$(
+  cat << 'END_OF_LINE'
+Set up the HTTP FreeIPA service on this host.
+
+Usage:
+  01_setup_http_service.sh [IPA-GROUP-ALLOWED-TO-CREATE-KEYTAB]
+  01_setup_http_service.sh (-h | --help)
```

Comment:
Instead of hard coding a filename. Removing the quotes on the here doc delimiter is necessary to have variable expansion.

```suggestion
  cat << END_OF_LINE
Set up the HTTP FreeIPA service on this host.

Usage:
  ${0##*/} [IPA-GROUP-ALLOWED-TO-CREATE-KEYTAB]
  ${0##*/} (-h | --help)
```